services/src/test-library/src/main/java/org/openkilda/testing/service/northbound/NorthboundService.java

```diff
@@ -110,6 +111,8 @@
 
     SwitchMeterEntries getAllMeters(SwitchId switchId);
 
+    SwitchValidationResult validateMeters(SwitchId switchId);
```

Comment:
It should be called `validateSwitch`, shouldn't it?

Reply:
Fixed